soweego/validator/checks.py

```diff
@@ -410,21 +540,27 @@ def links(
       of URL domains. Default: ``False``
     :param wd_cache: (optional) a ``dict`` of links gathered from Wikidata
       in a previous run. Default: ``None``
-    :return: 4 objects
+    :return: ``tuple`` of 6 objects
```

Comment:
a bit cumbersome to handle, consider using a NamedTuple or a dataclass

Reply:
Totally agree, filed in #422